ts/components/conversation/ConversationHeader.tsx

```diff
@@ -905,4 +908,4 @@ function OutgoingCallButtons({
     default:
       throw missingCaseError(outgoingCallButtonStyle);
   }
-}
+}
```

Comment:
The app's eslint style guide specifies trailing newlines. Can you revert this newline deletion?